sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -367,7 +367,14 @@ public IReadOnlyList<FormField> AsList()
 
             foreach (var kvp in _fieldValue.ValueObject)
             {
-                fieldDictionary[kvp.Key] = new FormField(kvp.Key, kvp.Value, _readResults);
+                if (kvp.Value == null)
+                {
+                    fieldDictionary[kvp.Key] = null;
```

Comment:
I need to check, but I think in Python we would still return a FormField here, but with null value so when a user tries to do a field["Tax"].value it wouldn't raise an attribute error. How does .NET handle this for other fields that are not found?

Reply:
I thought about it but what will the values for the properties on FormField be? In the case of .NET, `FormField.Value` can't be null as it is a struct and it was not set as nullable. I guess I can always make it nullable.... but that will be breaking.

>How does .NET handle this for other fields that are not found?

Could you expand more on this?

Reply:
> Could you expand more on this?

I was asking in a non-tables context. Say I had a label called "Fruit" and the service didn't extract a value for it on the form. What gets returned for this field? would field["Fruit] evaluate to null?

Reply:
If the service didn't extract a value for it, the label doesn't exist, right?
So in .NET the user always has to check first that the label (or dictionary key) exists.
For example in [invoice](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample13_RecognizeInvoicesFromFile.cs#L45),
```
if (invoice.Fields.TryGetValue("VendorName", out FormField vendorNameField))
{
    ...
}
```

But for a table, this is an interesting scenario... should we follow the same and hide the kay altogether? or have it visible but with a null value representing an empty cell. I still prefer the latter. What do you think?

Reply:
Let me check.... for invoice, yes, you're right the label doesn't exist. For labeled custom forms, I thought the behavior was different.

Reply:
Definitely latter IMO, and with the constraints on `FormField.Value` not being able to be set to null then this LGTM.

Reply:
For python, it's common to iterate through the dictionary like this: 

```
    for label, field in result.fields.items():
        print(label, field.value)
```

And so we will have an empty FormField with value=None here so the user doesn't accidentally encounter an AttributeError where an empty cell exists.

Reply:
Hmm yeah, that makes sense. I used to do it like that in .NET but with that struct there, it is no good :(
I know structs are good and everything but they are so painful too.